@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -68,9 +80,38 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
       _handleConnect: function () {
         this.state.rfbConnected = true
       },
-      _handleDisconnect: function () {
+      _handleDisconnect: async function () {
         this.state.rfbConnected = false
-        this.effects._connect()
+        const { protocol } = window.location
+        if (protocol === Protocols.https) {
+          try {
+            await fetch(`${protocol}//${this.state.url?.host}`)
+          } catch (error) {
+            console.error(error)
+            try {
+              await confirm({
+                icon: 'exclamation-triangle',
+                message: (
+                  <div>
+                    <p>
+                      <IntlMessage id='hostSeemsUnreachable' />
+                    </p>
+                    <a href={`${protocol}//${this.state.url?.host}`} rel='noopener noreferrer' target='_blank'>
+                      {this.state.url?.host}
+                    </a>
```

Comment:
Why do we need this?

Reply:
That to enhance the user experience. He just have to click on the link to be redirected to the concerned host